packages/generated-files-bot/src/generated-files-bot.ts

```diff
@@ -60,13 +60,20 @@ async function readExternalManifest(
   owner: string,
   repo: string
 ): Promise<Set<string>> {
-  const result = await github.repos.getContent({
-    owner,
-    repo,
-    path: manifest.file,
-  });
-  const content = Buffer.from(result.data.content, 'base64').toString();
-  return new Set(parseManifest(content, manifest.type, manifest.jsonpath));
+  return github.repos
+    .getContent({
+      owner,
+      repo,
+      path: manifest.file,
+    })
+    .then(result => {
+      const content = Buffer.from(result.data.content, 'base64').toString();
+      return new Set(parseManifest(content, manifest.type, manifest.jsonpath));
+    })
+    .catch(e => {
+      logger.warn(`error loading manifest: ${manifest.file}`);
```

Comment:
Should we log the error itself?  Wondering what we'll do with this warning without the exception.

Reply:
I think this is more of a configuration error - the configured manifest file doesn't exist. (in the errors we're currently getting, it's because some Java repos have completely custom READMEs so one of the manifest files does not exist).

Reply:
Also logging the error object itself at the warning level